modules/c++/xml.lite/unittests/test_Speed.cpp

```diff
@@ -0,0 +1,413 @@
+/* =========================================================================
+ * This file is part of xml.lite-c++
+ * =========================================================================
```

Comment:
File should be named `test_speed.cpp` (renamed when the class is renamed... lowercase snakecase for this file)